casper/src/test/resources/PoSTest.rho

```diff
@@ -443,12 +451,12 @@ match (
                 finalRewardsCh
             in {
               prepareUser!("9999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999", *setupCh) |
-              for (@validatorPubKey, _, _ <- setupCh) {
-                @PoS!("bond", 100, *setupCh) |
+              for (@validatorPubKey, _, _, @deployerId0 <- setupCh) {
```

Comment:
I would keep the names of the pubKey in sync with the names of the deployerId.

I'm thinking validatorDeployerId, user1Deployer1 and so on